.travis.yml

```diff
@@ -14,7 +14,7 @@ before_script:
   - . vendor/ci/travis/static-analysis.sh
 
 script:
-  - phpunit $PHPUNIT_FLAGS
+  - php vendor/bin/phpunit $PHPUNIT_FLAGS
```

Comment:
Sure but when the next version of PHPUnit comes out and travis updates it builds might stop working for you. (Well probably not because your tests are simple but you see the point.)

But I'll revert it if you want me to.